scripts/archive.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+source  /etc/profile.d/modules.sh
+
+set -a
+
+module purge
+module load pbs
+module use /g/data/hh5/public/modules
+module use ~access/modules
+module load cdo
+module load nco
+module load pythonlib/um2netcdf4/2.0
+module load conda/analysis3
+
+# set the environment variables for ACCESS-Archiver
+loc_exp=${PWD##*/}
+subdaily=false
+access_version=esmpayu
+arch_dir=/g/data/$PROJECT/$USER/archive/access-esm
+base_dir=/scratch/$PROJECT/$USER/access-esm/archive
```

Comment:
The payu archive directory isn't necessarily in `/scratch/$PROJECT/$USER/access-esm/archive`, it can be defined in `config.yaml` to be somewhere else, and quite often is in the case of users wanting to run their model with different project codes (where there are compute resources left), but read and write to the same laboratory space.

As this script requires for there to be an payu archive link (and directory) it would be more robust to query where `archive` is pointing to using
```
base_dir=$(readlink archive)
```

Reply:
ah I thought there must be a better way to do this! cheers :)